frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx

```diff
@@ -76,7 +76,7 @@ const InstallCluster: React.FC<InstallClusterProps> = ({ match }) => {
           setIndepModeSupportedPlatform(true);
         }
       });
-  });
+  }, []);
```

Comment:
Do we need useEffect if this is not going refreshed base on some parameters?

Reply:
Discussed in DMs